### DISCUSSION TITLE
Swedish translation

### DISCUSSION THREAD

**Discussion by eson57** [borgbase/vorta#891] [Translations]
Hi,
I have Swedish translation ready to go, if you want.
Please let me know where to upload, Transifex or Github is ok both, for me.

While testing I also found some missing/not working translation strings, seen in these dumps.

![screen_20210225-073513](https://user-images.githubusercontent.com/2996154/109116574-12fbbf00-7741-11eb-84e7-9e99c11c9f12.png)
![screen_20210225-074112](https://user-images.githubusercontent.com/2996154/109116601-1a22cd00-7741-11eb-9ecc-79cd2e98ed77.png)
![screen_20210225-074955](https://user-images.githubusercontent.com/2996154/109116614-1ee78100-7741-11eb-9de3-dd4b9f1f8af3.png)

**Comment by m3nu** [comment]
Fantastic! Just add it on Transifex, please. I'll look out for their notification to add the language and translator: https://www.transifex.com/borgbase/vorta/dashboard/

Regarding the missing strings, I'll add your screenshots as issue, so we can resolve this eventually.

**Comment by m3nu** [comment]
Added you on Transifex. Hope it worked.

**Reply by eson57** [reply]
Perfect! 👍🏻

**Comment by m3nu** [comment]
Just committing Swedish and updating all other complete langs

![Screen Shot 2021-02-26 at 10 03 23](https://user-images.githubusercontent.com/3916435/109244588-ec5d8700-7819-11eb-99f0-46bb250574f5.jpg)
